css/styles.css

```diff
@@ -120,7 +120,7 @@ textarea {
   border-radius: 5px;
   border: 0;
   color: white;
```

Comment:
please use only one kind of value for colors, use hex, or rgba or hardcoded color (orange, yellow, white)
using all in the same file will make it hard to maintain in the future

Reply:
OK!I integrated kinds of value for colors!

Reply:
I gave those colors the  same kind of values!